libr/core/cconfig.c

```diff
@@ -51,28 +51,40 @@ static void print_node_options(RConfigNode *node) {
 }
 
 static int compareName(const RAnalFunction *a, const RAnalFunction *b) {
-	return a && b && a->name && b->name && strcmp (a->name, b->name);
+	return (a && b && a->name && b->name ?  strcmp (a->name, b->name) : 0);
 }
 
 static int compareNameLen(const RAnalFunction *a, const RAnalFunction *b) {
-	return a && b && a->name && b->name && strlen (a->name) > strlen (b->name);
+	size_t la, lb;
+	if (!a || !b || !a->name || !b->name)
+		return 0;
+	la = strlen(a->name);
```

Comment:
and spaces after function
```suggestion
	la = strlen (a->name);
```

Reply:
Thanks, I did it for the other function, but forgot strlen.